Meta/Websites/serenityos.org/index.html

```diff
@@ -52,6 +52,8 @@ <h1>SerenityOS</h1>
     <li><a href="happy/2nd/">Happy 2nd birthday! SerenityOS: The second year</a></li>
     <li><a href="happy/1st/">Happy 1st birthday! SerenityOS: From zero to HTML in a year</a></li>
     <li><a href="https://happy-serenityos.linus.dev/">Linus's ":^)" tracker</a></li>
+    <li><a href="https://serenity.linus.dev/">VNC-access to Serenity running on Linus's hardware</a></li>
+    <li><a href="https://changelog.serenity/">Lubrsi's commit overview, grouped by month and category</a></li>
```

Comment:
I think you meant `https://changelog.serenityos.org`?

Reply:
Derp >_<